Easydict/NewApp/View/SettingView/Tabs/GeneralTab.swift

```diff
@@ -136,6 +136,16 @@ struct GeneralTab: View {
                 Toggle(isOn: $hideMenuBarIcon) {
                     Text("hide_menu_bar_icon")
                 }
+                #if !DEBUG
+                Picker(
+                    "modify_menu_bar_icon",
+                    selection: $selectedMenuBarIcon) {
+                        ForEach(MenuBarIconType.allCases, id: \.rawValue) { option in
+                            Image(option.imageName)
+                                .tag(option)
```

Comment:
If options conforms to the Identifiable, we don't need to add a tag.

https://developer.apple.com/documentation/swiftui/picker#Iterating-over-a-pickers-options

> [ForEach](https://developer.apple.com/documentation/swiftui/foreach) automatically assigns a tag to the selection views using each option’s id. This is possible because Flavor conforms to the [Identifiable](https://developer.apple.com/documentation/Swift/Identifiable) protocol.

Reply:
`tag` is for `Picker` instead of `ForEach`.